### ISSUE TITLE
Remove `.appengine.flex' bundle and clean things up once flex support goes live.

### ISSUE THREAD

**Issue by chanseokoh** [GoogleCloudPlatform/google-cloud-eclipse#1826]
`.appengine.flex` is meant to be a temporary bundle.

Things to clean up:
- The `.deploy.ui` bundle exports the package `.deploy.ui.flexible`, but that's just for exporting `FlexDeployCommandHandler` to the `.deploy.flex` bundle. The latter bundle will eventually go away, so remove the export declaration once that happens:
   ```
   Export-Package: com.google.cloud.tools.eclipse.appengine.deploy.ui.flexible
   ```
   Note that removing this now (and the relevant `Import-Package`) will not result in a compilation failure, but the command will fail at runtime.

**Comment by chanseokoh** [comment]
Another thing to clean up: some test bundles have the following in their `pom.xml`.
```
              <!-- TODO: remove this once the flex facet is fully available. -->
              <requirement>
                <type>eclipse-plugin</type>
                <id>com.google.cloud.tools.eclipse.appengine.flex</id>
                <versionRange>0.0.0</versionRange>
              </requirement>
```

**Comment by chanseokoh** [comment]
Closed #1826 via #1967.
